src/test/run-fail/meta-revision-bad.rs

```diff
@@ -13,10 +13,18 @@
 
 // revisions: foo bar
 // should-fail
-//[foo] error-pattern:bar
-//[bar] error-pattern:foo
+// [foo] error-pattern:bar
+// [bar] error-pattern:foo
```

Comment:
I think this may break these tests because compiletest currently searches for literally "//["